src/main/java/uk/gov/companieshouse/web/accounts/service/smallfull/impl/TangibleAssetsNoteServiceImpl.java

```diff
@@ -0,0 +1,126 @@
+package uk.gov.companieshouse.web.accounts.service.smallfull.impl;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.web.util.UriTemplate;
+import uk.gov.companieshouse.api.ApiClient;
+import uk.gov.companieshouse.api.error.ApiErrorResponseException;
+import uk.gov.companieshouse.api.handler.exception.URIValidationException;
+import uk.gov.companieshouse.api.model.accounts.smallfull.SmallFullApi;
+import uk.gov.companieshouse.api.model.accounts.smallfull.SmallFullLinks;
+import uk.gov.companieshouse.api.model.accounts.smallfull.tangible.TangibleApi;
+import uk.gov.companieshouse.web.accounts.api.ApiClientService;
+import uk.gov.companieshouse.web.accounts.exception.ServiceException;
+import uk.gov.companieshouse.web.accounts.model.smallfull.notes.tangible.TangibleAssets;
+import uk.gov.companieshouse.web.accounts.service.smallfull.TangibleAssetsNoteService;
+import uk.gov.companieshouse.web.accounts.transformer.smallfull.tangible.TangibleAssetsTransformer;
+import uk.gov.companieshouse.web.accounts.util.ValidationContext;
+import uk.gov.companieshouse.web.accounts.validation.ValidationError;
+
+@Service
+public class TangibleAssetsNoteServiceImpl implements TangibleAssetsNoteService {
+
+    @Autowired
+    private ApiClientService apiClientService;
+
+    @Autowired
+    private ValidationContext validationContext;
+
+    @Autowired
+    private TangibleAssetsTransformer tangibleAssetsTransformer;
+
+    private static final UriTemplate SMALL_FULL_URI =
```

Comment:
If you use the `getSmallFullAccounts` method in the `SmallFullService` this can go

Reply:
Resolved as it is inefficient to use the current `SmallFullService` method as it results in multiple ApiClient objects being created. This will likely be refactored in the future as it is not in the scope of the story.